crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -14,7 +14,42 @@
 # limitations under the License.
 #
 
+require "open3"
+
 module Api
-  class Upgrade
+  class Upgrade < Tableless
+    def status
+      {
+        crowbar: crowbar_upgrade_status,
+        checks: check
+      }
+    end
+
+    def check
+      {
+        sanity_checks: sanity_checks,
+        maintenance_updates_missing: maintenance_updates_missing?,
+        cluster_is_healthy: cluster_healthy?
+      }
```

Comment:
Hm, I know we've changed that before, but it would look better if all check functions would return true when we're OK. `maintenance_updates_missing` does the opposite...